docs/kubernetes/mailu/index.rst

```diff
@@ -0,0 +1,158 @@
+Install Mailu master on kubernetes
+==================================
+
+Prequisites
+-----------
+
+Structure
+~~~~~~~~~
+
+There’s chosen to have a double NGINX stack for Mailu, this way the main
+ingress can still be used to access other websites/domains on your
+cluster. This is the current structure:
+
+-  ``NGINX Ingress controller``: Listens to the nodes ports 80 & 443. We decided to keep the Ingress Controller, because he is good in what he does.
+-  ``Cert manager``: Creates automatic Lets Encrypt certificates based on an ``Ingress``-objects domain name.
+-  ``Mailu NGINX Front daemonset``: This daemonset runs in parallel with the Nginx Ingress Controller and only listens on all E-mail specific ports (25, 110, 143, 587,...)
+-  ``Mailu components``: All Mailu components are split into separate files to make them more handy to use
```

Comment:
- *We have* chosen to have a double NGINX stack for Mailu

Perhaps remove `We decided to keep the Ingress Controller, because he is good in what he does.`, it references some previous state and a later state. As a reader I have no clue about what was the state before.

- What are the mailu components? Those above?

Reply:
Done!